framework/output/RunOutputGroup.kt

```diff
@@ -212,7 +213,6 @@ internal class RunOutputGroup constructor(
 
     private suspend fun <T> consumeStreamResponse(
         stream: Response.Stream<T>,
-        onCompleted: (() -> Unit)? = null,
```

Comment:
This supposedly ran upon the graph building process completing, but actually run as soon as the graph building process had started (because `launchCompletableFuture`s don't wait for their completion). I've removed this as it did nothing and was a stumbling block whilst working on this.